### PR TITLE
added goBack onClick event on goback button

### PR DESCRIPTION
**Pull request by greentoss**
## Description

#1396 

tas: "follow the Constructor page. Click on Back button will redirect you to Main page, but "back" action is expected."

So the "back" action now works properly, it redirects to previous page
solution was made adding callback to BackButton in  ````ConstructorPage```` component
````<BackButton onClick={() => history.goBack()}/>````
without affecting on BackBotton component, adding new props there or changing config.routes

#### Screenshots
1
image
2
image
3
image

### Checklist

- [ ] 🔽 My branch is up-to-date with "development" branch
- [ ] ✅ All tests passed locally
- [ ] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [1396 ] 🔗 Link pull request to issue

### CODE REVIEW

**Review by byavdoshniak — APPROVED** [review]

**Review by tarasSanotskyi — APPROVED** [review]

**Review by byavdoshniak — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

File: src/pages/model/tests/constructor-page.test.js [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
 import { store, match } from './constructor-page.variables';
 import { config } from '../../../configs';
 
 const { constructorBasic } = config.labels.model;
+const mockUseHistory = jest.fn();
+const mockEditConstructor = jest.fn();
+
+jest.spyOn(console, 'error').mockImplementation(() => {});
+
+jest.mock('react-router-dom', () => ({
```
Comment (tarasSanotskyi) [inline src/pages/model/tests/constructor-page.test.js:15]: https://testing-library.com/docs/example-react-router/

File: src/pages/model/tests/constructor-page.test.js [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
+  ...jest.requireActual('react-router-dom'),
+  useHistory: () => ({
+    goBack: mockUseHistory
+  })
+}));
+
+jest.mock('react-redux', () => ({
```
Comment (tarasSanotskyi) [inline src/pages/model/tests/constructor-page.test.js:22]: https://redux.js.org/usage/writing-tests

**Review by MickhaelSh — DISMISSED** [review]

**Review by Pipuss — DISMISSED** [review]

File: src/pages/model/tests/constructor-page.test.js (hunk 1 above)
Comment (tarasSanotskyi) [inline src/pages/model/tests/constructor-page.test.js:15]: https://testing-library.com/docs/example-react-router/

File: src/pages/model/tests/constructor-page.test.js (hunk 2 above)
Comment (tarasSanotskyi) [inline src/pages/model/tests/constructor-page.test.js:22]: https://redux.js.org/usage/writing-tests#setting-up-a-reusable-test-render-function

**Review by Pipuss — APPROVED** [review]

**Pull request merged**